app/styleguide/routes/headings.vue

```diff
@@ -0,0 +1,135 @@
+<template>
+  <div :class="b()">
+    <div class="__example">
+      <h2>Headings</h2>
+
+      <h3>Headings</h3>
+      <div>
+        <e-heading
+          :underline="false"
+          tag-name="h1">
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor
+        </e-heading>
+      </div>
+      <div>
+        <e-heading
+          :underline="false"
```

Comment:
- [ ] Use `boolean` props without value. This way if they are defined, they will be `true`, if missing `false`.